apps/projects/contracts/Projects.sol

```diff
@@ -403,7 +407,7 @@ contract Projects is AragonApp, DepositableStorage {
 ///////////////////
 
     /**
-     * @notice Request assignment for issue `_issueNumber`
+     * @notice Request assignment for issue `_issueNumber` with application `_application`
```

Comment:
Is the IPFS hash of their application something the user will recognize (in which case it makes sense to show it here), or something we generate in the background?

Reply:
We may have to just put it in the docs so users know it's an ipfs hash. Ideally the client should be able to determine an IPFS hash based on it starting with a Q and it being a certain number of characters, and generate a link that is tied to the client's gateway

Reply:
For now giving something is better than the users having no data at all in the voting app? i think?